modules/gdscript/gdscript_tokenizer.cpp

```diff
@@ -135,7 +135,8 @@ const char *GDScriptTokenizer::token_names[TK_MAX] = {
 	"NAN",
 	"Error",
 	"EOF",
-	"Cursor"
+	"Cursor",
+	"##" //tooltip
```

Comment:
Add a trailing comma for easier refactoring, and tweak the comment style to match what we use now:

```suggestion
	"##", // Tooltip comment for exported variables.
```